src/evaluator/expression-evaluator.ts

```diff
@@ -376,19 +390,31 @@ export class ExpressionEvaluator {
       throw new ExpressionError(`Cannot index with type ${typeof property}`);
     }
 
-    if (typeof value.value === 'object') {
-      if (value.value.hasOwnProperty(property.value)) {
-        return {
-          value: (value.value as any)[property.value],
-          nodes: 1 + value.nodes + property.nodes,
-          functionCalls: value.functionCalls + property.functionCalls,
-        };
-      } else {
-        throw new ExpressionError(`Value does not have property ${property}`);
+    if (this._memberChecks) {
+      for (const checkFn of this._memberChecks) {
+        if (checkFn(value.value, property.value)) {
```

Comment:
Wouldn't this better be called isMemberFunction?

Reply:
I don't think the name here matters all that greatly, but I'll adjust it to `memberCheckFn`, since that follows the config property name.

Reply:
You've gotten too Rusty ;)

Reply:
`fn` is standard and i've used it for years :triumph: